### PR TITLE
katacoda: Small cleanup and added playground demo.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: tutorials/katacoda/thanos/x-secret-rawkode/intro.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Demo Resources
+
+Let's go!
+
+### Feedback
+
```
Comment (yeya24) [inline tutorials/katacoda/thanos/x-secret-rawkode/intro.md]: Do we still need to keep this secret demo?

Comment (yeya24) [reply tutorials/katacoda/thanos/x-secret-rawkode/intro.md]: Nvm seems this is removed already.

Comment (bwplotka) [reply tutorials/katacoda/thanos/x-secret-rawkode/intro.md]: Yes (: I moved it, thanks for looking!

**Comment by bwplotka** [comment]
@yeya24 Thanks for looking. I finished tweaking, all should be mergable!

**Comment by bwplotka** [comment]
Merging for KubeCon office hours :hugs:

File: tutorials/katacoda/thanos/x-playground/step2.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+In this step, we will configure the object store and change sidecar to upload to the
+object-store.
+
+## Running Minio
+
+```
+mkdir ${CURR_DIR}/minio && \
+docker run -d --rm --name minio \
+     -v ${CURR_DIR}/minio:/data \
+     -p 9000:9000 -e "MINIO_ACCESS_KEY=rawkode" -e "MINIO_SECRET_KEY=rawkodeloveobs" \
```
Comment (yeya24) [inline tutorials/katacoda/thanos/x-playground/step2.md:14]: Do we need to change the key?

**Pull request merged**